grpc/server/CMakeLists.txt

```diff
@@ -27,10 +29,39 @@ add_custom_command(OUTPUT dronecore.pb.cc
     DEPENDS ${proto_dir}/dronecore.proto
 )
 
+
+foreach(plugin ${plugins_list})
+    add_custom_command(OUTPUT ${plugin}.grpc.pb.cc
+        COMMAND ${Protobuf_PROTOC_EXECUTABLE}
+            -I ${proto_dir}
+            -I ${plugins_dir}/${plugin}
+            --grpc_out=.
+            --plugin=protoc-gen-grpc=${GRPC_CPP_PLUGIN}
+            --cpp_out=. ${plugins_dir}/${plugin}/${plugin}.proto
+        DEPENDS ${plugins_dir}/${plugin}/${plugin}.proto
+)
+endforeach()
+
+foreach(plugin ${plugins_list})
+    add_custom_command(OUTPUT ${plugin}.pb.cc
+        COMMAND ${Protobuf_PROTOC_EXECUTABLE}
+            -I ${proto_dir}
+            -I ${plugins_dir}/${plugin}
+            --cpp_out=. ${plugins_dir}/${plugin}/${plugin}.proto
+        DEPENDS ${plugins_dir}/${plugin}/${plugin}.proto
+)
+endforeach()
+
 add_executable(dronecore_server
     dronecore_server.cpp
     dronecore.pb.cc # generated
     dronecore.grpc.pb.cc # generated
+	action.pb.cc # generated
+    action.grpc.pb.cc # generated
+    mission.pb.cc # generated
+    mission.grpc.pb.cc # generated
+    telemetry.pb.cc # generated
+    telemetry.grpc.pb.cc # generated
```

Comment:
You could also generate that list using the same `foreach`. You just add it to all to a variable.